examples/mongodbatlas_advanced_cluster/tenant-upgrade/main.tf

```diff
@@ -9,8 +9,6 @@ resource "mongodbatlas_advanced_cluster" "cluster" {
   cluster_type = "REPLICASET"
 
   replication_specs {
-    num_shards = 1
-
```

Comment:
`num_shards` is deprecated and can be removed here without any other changes as value is 1.